src/UI/class-settings-page.php

```diff
@@ -941,6 +956,16 @@ public function validate_options( array $input ): array {
 			$input['disable_javascript'] = 'true' === $input['disable_javascript'];
 		}
 
+		if ( 'true' !== $input['disable_autotrack'] && 'false' !== $input['disable_autotrack'] ) {
+			add_settings_error(
+				Parsely::OPTIONS_KEY,
+				'disable_autotrack',
+				__( 'Value passed for disable_autotrack must be either "true" or "false".', 'wp-parsely' )
```

Comment:
The visible option are `Yes` and `No`, so having the value set as true/false, and then reporting back on that, may be confusing for the user. Make the labels better match the values, and then update the validation message.